Description: Added necessary <stdio.h> header
Author: Ervin Hegedus <airween@gmail.com>
Bug-Debian: https://bugs.debian.org/1067364

--- libnginx-mod-http-modsecurity-1.0.3.orig/config
+++ libnginx-mod-http-modsecurity-1.0.3/config
@@ -10,7 +10,8 @@
 
 ngx_feature_name=
 ngx_feature_run=no
-ngx_feature_incs="#include <modsecurity/modsecurity.h>"
+ngx_feature_incs="#include <modsecurity/modsecurity.h>
+#include <stdio.h>"
 ngx_feature_libs="-lmodsecurity"
 ngx_feature_test='printf("hello");'
 ngx_modsecurity_opt_I=
